docoptTests/DocoptTestCasesTests.swift

```diff
@@ -65,7 +60,12 @@ class DocoptTestCasesTests: XCTestCase {
     
     private func fixturesFileContents() -> String {
         if let filePath = self.fixturesFilePath() {
-            let fileContents = String(contentsOfFile: filePath, encoding: NSUTF8StringEncoding, error: nil)
+            let fileContents: String?
+            do {
+                fileContents = try String(contentsOfFile: filePath, encoding: NSUTF8StringEncoding)
+            } catch _ {
+                fileContents = nil
+            }
             return fileContents!
```

Comment:
we also know this constructor should never return nil, so i've cleaned this up with try! as well.